src/screens/Sidebar/SidebarScreen.js

```diff
@@ -111,90 +116,88 @@ class SidebarScreen extends React.Component {
     ];
     return (
       <View style={{ backgroundColor: colors.listBackPurple }}>
-        <View>
-          <ProgressiveImage
-            hasOverlay
-            style={styles.headerCoverImage}
-            source={{ uri: 'https://fubukinofansub.files.wordpress.com/2011/12/cover-03-04.jpg' }}
+        <ProgressiveImage
+          hasOverlay
+          style={styles.headerCoverImage}
+          source={{ uri: 'https://fubukinofansub.files.wordpress.com/2011/12/cover-03-04.jpg' }}
```

Comment:
is this temporary? if not then we should store this in the build so we don't have to make a network request.

Reply:
temporary!